katportalclient/test/test_client.py

```diff
@@ -1360,23 +1363,25 @@ def test_sensor_subarray_katcp_name_lookup(self):
                 invalid_response=['error'],
                 starts_with=lookup_base_url,
                 contains=sensor_name_filter)
-        sensor = yield self._portal_client.sensor_subarray_lookup('cbf', sensor_name_filter, True)
+        sensor = yield self._portal_client.sensor_subarray_lookup(
+                'cbf', sensor_name_filter, True)
         self.assertTrue(sensor == expected_sensor_name)
 
-
     @gen_test
     def test_sensor_subarray_invalid_sensor_lookup(self):
         """Test that sensor subarray lookup can correctly handle an invalid sensor name."""
-        lookup_base_url = self._portal_client.sitemap['subarray'] + '/3/sensor-lookup/anc/device_status/0'
+        lookup_base_url = (self._portal_client.sitemap['subarray'] + 
+        '/3/sensor-lookup/anc/device_status/0')
         sensor_name_filter = 'device_status'
         self.mock_http_async_client().fetch.side_effect = mock_async_fetcher(
-                valid_response='{"error":"SensorLookupError: Could not lookup the sensor on component. Could not determine component."}',
+                valid_response='{"error":"SensorLookupError: Could not lookup the sensor '
+                'on component. Could not determine component."}',
                 invalid_response='[]',
                 starts_with=lookup_base_url,
                 contains=sensor_name_filter)
         with self.assertRaises(SensorLookupError):
-            yield self._portal_client.sensor_subarray_lookup('anc', sensor_name_filter, False)
-
```

Comment:
Removing this blank line wasn't necessary, as we have a function defined next (not a method).  We need two lines before a function.  These kind of rules are hard to remember, so I like my editor to remind me.  If you have an editor configured to show you flake8 warnings, it is really easy to spot.  E.g., using the Atom editor with the flake8 linter plugin:

![image](https://user-images.githubusercontent.com/20420754/33262928-9c645360-d370-11e7-9042-61c055affeea.png)

Similar things can be done in vi.